parts/26-report.sh

```diff
@@ -623,7 +626,7 @@ report_data-info(){
 object_store_to_path=$(cat <<EOF
 import sys
 import xml.etree.ElementTree as et
-config_file = "$2"
+config_file = "$arg_config_file"
```

Comment:
if this isn't provided, what happens? Does it fail gracefully or is there an unhelpful error message? Maybe this variable needs to be guarded in `if [[ -n "$object_store_config_file" ]]`?

```suggestion
config_file = "$arg_object_store_config_file"
```

the argument name is taken directly from the parts after `##?`.

Reply:
I'm not sure if I understood it correctly but modified accordingly. I tested this by pasting this function into my gxadmin script and it failed to detect the variables. Am I doing something wrong here?